src/docfx/lib/git/CommitBuildTimeProvider.cs

```diff
@@ -34,14 +34,8 @@ public CommitBuildTimeProvider(Config config, Repository repo)
             _buildTimeByCommit = commitBuildTime.Commits.ToDictionary(item => item.Sha, item => item.BuiltAt);
         }
 
-        public bool TryGetCommitBuildTime(string commitId, out DateTime time)
-        {
-            if (!_buildTimeByCommit.TryGetValue(commitId, out time))
-            {
-                time = _buildTime;
-            }
-            return true;
-        }
+        public DateTime GetCommitBuildTime(string commitId)
+            => _buildTimeByCommit.TryGetValue(commitId, out var time) ? time : _buildTime;
```

Comment:
Since I did not change the build history cache working mechanism, did not think of this before.
To me, it seems like it would be different for the first time build on different instances.

Reply:
The build history cache will be uploaded to be shared for all the build instances

Reply:
Can you help me get into my outlook?